test/rfq_maker_balance_cache_test.ts

```diff
@@ -0,0 +1,120 @@
+import { BalanceCheckerContract } from '@0x/asset-swapper';
+import { artifacts } from '@0x/asset-swapper/lib/src/artifacts';
+import { artifacts as erc20Artifacts, DummyERC20TokenContract } from '@0x/contracts-erc20';
+import { expect } from '@0x/contracts-test-utils';
+import { web3Factory, Web3ProviderEngine } from '@0x/dev-utils';
+import { BigNumber } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import 'mocha';
+import { Connection, Repository } from 'typeorm';
+
+import * as config from '../src/config';
+import { getDBConnectionAsync } from '../src/db_connection';
+import { MakerBalanceChainCacheEntity } from '../src/entities';
+import { cacheRfqBalancesAsync } from '../src/runners/rfq_maker_balance_cache_runner';
+
+import { setupDependenciesAsync, teardownDependenciesAsync } from './utils/deployment';
+
+const SUITE_NAME = 'RFQ Maker Balance Cache Test';
+
+describe(SUITE_NAME, () => {
+    let provider: Web3ProviderEngine;
+    let balanceCheckerContract: BalanceCheckerContract;
+    let dbConnection: Connection;
+    let zrx: DummyERC20TokenContract;
+    let balanceRepo: Repository<MakerBalanceChainCacheEntity>;
+    let web3Wrapper: Web3Wrapper;
+    let makerAddress1: string;
+    let makerAddress2: string;
+
+    before(async () => {
+        await setupDependenciesAsync(SUITE_NAME);
+
+        // connect to ganache
+        const ganacheConfigs = {
+            shouldUseInProcessGanache: false,
+            shouldAllowUnlimitedContractSize: true,
+            rpcUrl: config.ETHEREUM_RPC_URL,
+        };
+
+        provider = web3Factory.getRpcProvider(ganacheConfigs);
+
+        web3Wrapper = new Web3Wrapper(provider);
+
+        const accounts = await web3Wrapper.getAvailableAddressesAsync();
+        const deployer = accounts[0];
+        makerAddress1 = accounts[1];
+        makerAddress2 = accounts[2];
+
+        zrx = await DummyERC20TokenContract.deployFrom0xArtifactAsync(
+            erc20Artifacts.DummyERC20Token,
+            provider,
+            { from: deployer, gas: 10000000 },
+            {},
+            '0x Protocol Token',
+            'ZRX',
+            new BigNumber(18),
+            new BigNumber(1000000),
+        );
+
+        await zrx.mint(new BigNumber(100)).awaitTransactionSuccessAsync({ from: makerAddress1 });
```

Comment:
I know this may not change anything, but we may want to simply keep the decimals into consideration here. You are minting 1e-16 tokens. Did you want to mint `100000000000000000000` tokens?

```typescript
Web3Wrapper.toBaseUnitAmount(100, 18)
```

Reply:
same comment for below

Reply:
the balance pipeline never deals with decimals, so I'm not too worried it. I think prioritizing simplicity here is better